crates/bevy_render/src/view/mod.rs

```diff
@@ -802,12 +810,22 @@ pub fn prepare_view_targets(
     manual_texture_views: Res<ManualTextureViews>,
 ) {
     let mut textures = HashMap::default();
+    let mut output_textures = HashMap::default();
     for (entity, camera, view, texture_usage) in cameras.iter() {
         if let (Some(target_size), Some(target)) = (camera.physical_target_size, &camera.target) {
```

Comment:
I think some of the nesting here could be reduced with some let-else

Reply:
Thanks this helped a lot.